install/check-minimum-requirements.sh

```diff
@@ -26,6 +26,14 @@ if [[ -z "$COMPOSE_VERSION" ]]; then
   exit 1
 fi
 
+STANDALONE_COMPOSE_VERSION=$($dc_base_standalone version --short &>/dev/null || echo '')
+if [[ ! -z "${STANDALONE_COMPOSE_VERSION}" ]]; then
+  if [[ "$(vergte ${COMPOSE_VERSION//v/} ${STANDALONE_COMPOSE_VERSION//v/})" -eq 1 ]]; then
+    COMPOSE_VERSION="${STANDALONE_COMPOSE_VERSION}"
+    dc_base='docker-compose'
+  fi
+fi
```

Comment:
I'd say we should move this logic to `install/dc-detect-version.sh`. What do you think?

Reply:
Definitely, it would make sense there.

Reply:
`COMPOSE_VERSION` was defined in `check-minimum-requirements.sh`, that's why I had added `STANDALONE_COMPOSE_VERSION` there as well.

But you're right, I moved both and `vergte` function to `dc-detect-version.sh`.